rapidtide/correlate.py

```diff
@@ -85,6 +73,8 @@ def resdec(f):
 
 
 def conditionaljit2():
```

Comment:
This function isn't used within this module, and is defined within any module where it _is_ used.

Reply:
Ok to remove it then.